packages/backend/src/plugins/k8s-apply.ts

```diff
@@ -125,16 +121,18 @@ export const createKubernetesApply = (config: Config) => {
             },
           ],
         };
-        if (!confFile.clusters[0].cluster['insecure-skip-tls-verify']) {
+        if(!confFile.clusters[0].cluster['insecure-skip-tls-verify']) {
           let caDataRaw = targetCluster.getOptionalString('caData');
           if (caDataRaw?.startsWith('-----BEGIN CERTIFICATE-----')) {
             caDataRaw = Buffer.from(
               targetCluster.getString('caData'),
               'utf8',
             ).toString('base64');
           }
-          confFile.clusters[0].cluster['certificate-authority-data'] =
+          (confFile.clusters[0].cluster as any)['certificate-authority-data'] =
             caDataRaw;
+          (confFile.clusters[0].cluster as any)['certificate-authority'] =
+            targetCluster.getOptionalString('caFile');
```

Comment:
We should check for presence of this value to avoid having an empty field. Similar to what you are experiencing right now.

Reply:
Made changes as per your suggestions @nabuskey